### ISSUE TITLE
Subscription popup looks bad on Bedrock theme

### ISSUE THREAD

**Issue by pkuliga** [Automattic/jetpack#35413]
### To reproduce
1. Enable the Subscription popup from the Newsletter Settings
2. Use the Bedrock theme
3. Open the post and scroll down to open the popup
4. Notice that the text above the input is invisible

![Image](https://github.com/Automattic/jetpack/assets/4068554/1b639831-dd7c-4049-95ac-07926b9d4e82)

### Expected result

The text above the input is visible.

**Comment by rickmgithub** [comment]
## 📌 REPRODUCTION RESULTS ##
- Tested on Simple – Replicated 
- Tested on Atomic – Replicated  

## 📌 FINDINGS/SCREENSHOTS/VIDEO ##
Exactly as the OP said.
There is a workaround and that is to edit the popup button to give it black letters but as the view in the editor then show black letters on a black background while the popup is white, it's confusing for users. 

## 📌 ACTIONS ##
Triaged

**Comment by bizanimesh** [comment]
👋 I can reproduce the issue ([screenshot](https://github.com/Automattic/jetpack/assets/52444539/7e4740a6-f18a-46f4-bace-86a04d96ebd0)) - I see because of white color the text is not visible. But this can be modified from the theme editor.

@pkuliga - have you tried editing the pop-up block? You can go to your WordPress.com dashboard and change the color of the texts like this (as a workaround):

https://github.com/Automattic/jetpack/assets/52444539/e07796ed-0dcf-49e0-8c80-f60bf773b574

**Comment by pkuliga** [comment]
@bizanimesh yup, I know I can edit the template but shouldn't the text be visible by default?

Modal already inherits some of the theme styles, like text color, but the background is always white. That's why we have a problem.

I think we need to either inherit the theme background color as well, if possible, to make the text readable or to not inherit the text color and define our own that will be readable on a white background.

**Comment by bizanimesh** [comment]
Hello @pkuliga - yes, the team will take a look. In the meantime, you may apply the workaround. :)
